server/market/bookrouter.go

```diff
@@ -16,14 +16,14 @@ import (
 	"decred.org/dcrdex/server/comms"
 )
 
-// A bookUpdateAction classifies updates into how they affect the book or epoch
+// A updateAction classifies updates into how they affect the book or epoch
 // queue.
-type bookUpdateAction uint8
+type updateAction uint8
```

Comment:
This name is not exported, and the purpose is broader than book/unbook these days, although the target is the *book* router.  I just felt this was a less confusing name given the use of the channel for these sigs.